src/js/interface.js

```diff
@@ -456,6 +459,16 @@ window.addEventListener("load", function() {
                 if (value===10) {
                   return document.createElement("br");
                 } else {
+                  if(value>=-10240 && value<=-9217) {
+                    convertOutput.highSurrogate = value;
+                    return document.createTextNode("");
+                  }
+                  if(convertOutput.highSurrogate!=0 &&
+                     value>=-9216 && value<=-8193) {
+                       var output = String.fromCharCode(convertOutput.highSurrogate, value);
+                       convertOutput.highSurrogate = 0;
```

Comment:
You may also want to reset the high surrogate value if the value it receives is not a low surrogate value.

Reply:
Going against my own request here... of course you're right.

As far as resetting the high surrogate value goes, I thought that just discarding the input via the default line would be sufficient, but I agree that your suggestion certainly is cleaner.

Reply:
I haven't even looked at the input code at all to see how to handle this. I'll give it a try.